pkgs/development/compilers/djgpp/default.nix

```
@@ -0,0 +1,137 @@
+{ bison, buildPackages, curl, file, fetchFromGitHub, fetchurl, flex, i686Build ? false
+, lib, makeWrapper, perl, stdenv, texinfo, unzip, which }:
+
+stdenv.mkDerivation rec {
+  pname = "djgpp";
+  version = "12.1.0";
+  buildVersion = if i686Build then "${version}-i686" else "${version}";
+
+  # adapted from https://github.com/andrewwutw/build-djgpp/blob/master/script/12.1.0
+  DJGPP_DOWNLOAD_BASE = "https://www.mirrorservice.org/sites/ftp.delorie.com/pub";
+  FTPMIRROR_GNU_DOWNLOAD_BASE = "https://www.mirrorservice.org/sites/ftp.gnu.org/gnu";
+  FTP_GNU_DOWNLOAD_BASE = "https://www.mirrorservice.org/sites/ftp.gnu.org/gnu";
+  BINUTILS_VERSION = "230";
+  DJCRX_VERSION = "205";
+  DJLSR_VERSION = "205";
+  DJDEV_VERSION = "205";
+  GCC_VERSION = "12.1.0";
+  GCC_VERSION_SHORT = "12.10";
+  GMP_VERSION = "6.2.1";
+  MPFR_VERSION = "4.1.0";
+  MPC_VERSION = "1.2.1";
+  AUTOCONF_VERSION = "2.69";
+  AUTOMAKE_VERSION = "1.15.1";
+
```

```suggestion
};

```

(And indent it after)